src/SWP/Bundle/UserBundle/Resources/config/routing.yml

```diff
@@ -1,3 +1,7 @@
 swp_user:
     resource: "@SWPUserBundle/Controller"
-    type:     annotation
\ No newline at end of file
+    type:     annotation
+
+fos_user_register:
+    resource: "@FOSUserBundle/Resources/config/routing/registration.xml"
+    prefix: /register
```

Comment:
new line is missing

so there is also an option to register/login via normal Symfony form?